dali/python/nvidia/dali/plugin/tf.py

```diff
@@ -43,6 +44,14 @@
 
 _dali_tf = _dali_tf_module.dali
 
+_dali_tf.__doc__ = _dali_tf.__doc__ + """
+
+    WARNING: 
+    -------
+    Please keep in mind that TensorFlow allocates almost all available device memory by default. This might cause errors in 
+    DALI due to insufficient memory.
```

Comment:
I would add a hint how to limit TF hunger for the memory.

Reply:
Done, but a added the hint to look into the TF docs, as this may vary depanding on the use case: version of TensorFlow, API that is used (like tf.keras or session).

The most general way is to set up virtual device:
```python
gpus = tf.config.experimental.list_physical_devices('GPU')
if gpus:
  # Restrict TensorFlow to only allocate 1GB of memory on the first GPU
  try:
    tf.config.experimental.set_virtual_device_configuration(
        gpus[0],
        [tf.config.experimental.VirtualDeviceConfiguration(memory_limit=1024)])
    logical_gpus = tf.config.experimental.list_logical_devices('GPU')
    print(len(gpus), "Physical GPUs,", len(logical_gpus), "Logical GPUs")
  except RuntimeError as e:
    # Virtual devices must be set before GPUs have been initialized
    print(e)
```
but I wouldn't like to include this into our docs. 

Anyway, it's easy to google how to do this.

Reply:
Sure